File: servers/node-expressjs/Benchmark.js
var express = require("express");
const fs = require('fs');

var app = express();

var fib = function (n) {
    if (n === 0) {
        return 0;
    } else if (n == 1) {
        return 1;
    } else {
        return fib(n - 1) + fib(n - 2)
    }
};

app.get("/fib/:number", function (req, res) {
    var number = req.param("number");
    var result = fib(number);
    res.send("___Node+Express: fib(" + number + ") = " + result);
});

app.get("/file", function (req, res) {
    const path = '../sample/commons-lang-2.6.jar'
    //res.sendFile(path)
    const src = fs.createReadStream(path);
    src.pipe(res);
})

app.listen(3000);

File: servers/python-flask/benchmark.py
from flask import Flask
from flask import send_file

app = Flask(__name__)

@app.route('/fib/<int:number>')
def index(number=1):
    return "___Python+Flask: fib("+ str(number) + ") = " + str(fib(number))

@app.route('/file')
def send_jar():
    try:
        return send_file('../sample/commons-lang-2.6.jar', attachment_filename='python.jpg')
    except Exception as e:
        return str(e)

def fib(n):
    if n == 0:
        return 0
    elif n == 1:
        return 1
    else:
        return fib(n-1) + fib(n-2)

if __name__ == '__main__':
    app.run(debug=False)